cli/algorithms.go

```diff
@@ -19,6 +19,7 @@ var (
 	pythonRuntime    string
 	algoType         string
 	requirementsFile string
+	algoArgs         []string
```

Comment:
I have a wasm module that can be run by

```bash
wasmtime run ./iris-inference.wasm '{"sepal_length": 7.0, "sepal_width": 3.2, "petal_length": 4.7, "petal_width": 1.4}'
```

I am unable to upload the algorithm with `'{"sepal_length": 7.0, "sepal_width": 3.2, "petal_length": 4.7, "petal_width": 1.4}'` as an argument

```bash
./build/cocos-cli algo ./iris-inference.wasm ./private.pem -a wasm --args="{\"sepal_length\": 7.0, \"sepal_width\": 3.2, \"petal_length\": 4.7, \"petal_width\": 1.4}"
Error: invalid argument "{\"sepal_length\": 7.0, \"sepal_width\": 3.2, \"petal_length\": 4.7, \"petal_width\": 1.4}" for "--args" flag: parse error on line 1, column 2: bare " in non-quoted-field
```

```bash
./build/cocos-cli algo ./iris-inference.wasm ./private.pem -a wasm --args="'{\"sepal_length\": 7.0, \"sepal_width\": 3.2, \"petal_length\": 4.7, \"petal_width\": 1.4}'"
Error: invalid argument "'{\"sepal_length\": 7.0, \"sepal_width\": 3.2, \"petal_length\": 4.7, \"petal_width\": 1.4}'" for "--args" flag: parse error on line 1, column 3: bare " in non-quoted-field
```

```bash
./build/cocos-cli algo ./iris-inference.wasm ./private.pem -a wasm --args='{"sepal_length": 7.0, "sepal_width": 3.2, "petal_length": 4.7, "petal_width": 1.4}'
Error: invalid argument "{\"sepal_length\": 7.0, \"sepal_width\": 3.2, \"petal_length\": 4.7, \"petal_width\": 1.4}" for "--args" flag: parse error on line 1, column 2: bare " in non-quoted-field
```

Reply:
is it possible to take non json args

Reply:
alternatively we can add a json-args flag

Reply:
something like `go run ./cmd/cli/main.go algo ./test/manual/algo/addition.py private.pem -a python --args sepal_length=7.0,sepal_width=3.2,petal_length=4.7,petal_width=1.4`

Reply:
resolved this should be working now